cpp/Platform.Converters/CachingConverterDecorator.h

```diff
@@ -1,15 +1,28 @@
-﻿namespace Platform::Converters
+﻿#pragma once
+#include <functional>
+#include <memory>
+#include <unordered_map>
+
+namespace Platform::Converters
 {
```

Comment:
As long as this is no longer the Decorator template we can rename this class to just Cached.